java/src/com/ibm/streamsx/topology/internal/context/remote/ZippedToolkitRemoteContext.java

```diff
@@ -117,6 +117,9 @@ private static Path pack(final File appTkRoot, JsonObject submission, String tkN
         
         // Paths to completely copy into the code archive
         Map<Path, String> paths = new HashMap<>();
+
+        Map<Path, String> SPLMM_toolkits = new HashMap<>();
+        String[] splmm_dir = {""};
```

Comment:
Why is this an array?

Reply:
For line 121, I mimicked how toolkits were created using a map, and for line 122, I couldn't use a regular string for the splmm dir, bc where I set the value (in the lambda function on line 154) if I use a regular string i get the java error 'local variables referenced from a lambda expression must be final or effectively final', and one of the answers stated that a String array is a workaround.

Reply:
Ok - makes sense but the typical value for not set would be `null`, not an empty string.

Reply:
I will change it to null and re-run the tests

Reply:
Fixed in latest commit